src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -1378,9 +1378,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return _lazyLexicalSortKey
         End Function
 
+        Public Function LocationKey(ByVal l As Location) As Long
```

Comment:
Consider adding a comment about how do we fill certain bits in the key so that numbers 62 and 40 didn't look like something random.